app/src/main/AndroidManifest.xml

```diff
@@ -18,6 +18,10 @@
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
         </activity>
+        <meta-data
+            android:name="io.fabric.ApiKey"
+            android:value="e5627bac828b2b0b56a64f44f0ffba67f68324fa"
```

Comment:
in that case, whoever will run the sample and make changes that will cause a crash also we will be notified.

would be better to be something in ignore right?